trace-sdk/src/androidTest/java/io/bitrise/trace/data/collector/device/DeviceNetworkTypeDataCollectorInstrumentedTest.java

```diff
@@ -1,28 +1,40 @@
 package io.bitrise.trace.data.collector.device;
 
+import androidx.annotation.NonNull;
+import androidx.test.filters.SdkSuppress;
+
 import org.junit.Test;
 
-import io.bitrise.trace.data.dto.Data;
 import io.bitrise.trace.data.collector.BaseDataCollectorInstrumentedTest;
+import io.bitrise.trace.data.dto.Data;
 
+import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
 
 /**
  * Instrumented tests for {@link DeviceNetworkTypeDataCollector}.
  */
 public class DeviceNetworkTypeDataCollectorInstrumentedTest extends BaseDataCollectorInstrumentedTest {
 
-    /**
-     * Verifies that when {@link DeviceNetworkTypeDataCollector#collectData()} is called, the content of the returned
-     * {@link Data} should not be {@code null}.
-     */
     @Test
-    public void collectData_contentShouldBeNotNull() {
-        final DeviceNetworkTypeDataCollector deviceNetworkTypeDataCollector =
-                new DeviceNetworkTypeDataCollector(context);
-        final String actualValue = (String) deviceNetworkTypeDataCollector.collectData().getContent();
-        assertThat(actualValue, is(notNullValue()));
+    @SdkSuppress(maxSdkVersion = 28)
+    public void collectData_pre29() {
+        final DeviceNetworkTypeDataCollector collector = new DeviceNetworkTypeDataCollector(context);
+        assertNetworkCollectedData(collector.collectData(), "UNKNOWN");
+    }
+
+    @Test
+    @SdkSuppress(minSdkVersion = 29)
+    public void collectData_post29() {
```

Comment:
nit: post stands fro after, but 29 is also included.

Reply:
updated to read `collectData_sdk29AndAfter` and `collectData_preSdk29` - does that make it clearer?